packages/dev/core/src/Layers/layer.ts

```diff
@@ -35,6 +35,20 @@ export class Layer {
      */
     public isBackground: boolean;
 
+    private _applyPostProcess: boolean = true;
+    /**
+     * If scene post processes should affect this Layer.
+     * Is always true if the layer is a foreground

```

Comment:
I think you meant the other way around? meaning, "Is always true if the layer is a background layer"?

Reply:
Thanks for catching that! I initially wrote the property the other way around but forgot to update the docs 🤣